packages/react/src/components/DetailsList/DetailsList.base.tsx

```diff
@@ -1255,6 +1270,14 @@ export class DetailsListBase extends React.Component<IDetailsListProps, IDetails
         calculatedWidth: column.minWidth || MIN_COLUMN_WIDTH,
       };
 
+      //save new keyboard input resize value to columnOverrides
+      if (
```

Comment:
This is the kind of logic which concerns me. You're counting on this `keyboardColumnEditProps` being set and unset. That's very expensive with respect to render performance, and also very brittle. We already have an internal handler for `onColumnResized` that we should just invoke, and there's an external handler for `columnReorderProps.onDragEnd` we obviously need to invoke if the order is changed by the updates.